src/Renderer/OrientedImageCamera.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from 'three';
+
+class OrientedImageCamera extends THREE.PerspectiveCamera {
```

Comment:
We need more information on the purpose of this new camera, please add some documentation.

Reply:
I continued working on this camera after handing it to @gliegard and my version has a better documentation that you can adapt/reuse : https://github.com/mbredif/three-photogrammetric-camera/blob/master/src/cameras/PhotogrammetricCamera.js

Reply:
ok, I've add documentation.
Thanks Mathieu.